### PR TITLE
fix(DB/Creature): Port Treebole WP from mangos-wotlk

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4996
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15007

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

mangos-wotlk DB

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1719322993021854900.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+DELETE FROM `waypoint_data` WHERE `id` = 222150;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(222150, 1, 3609.03, 6829.57, 136.69, 5.2105, 300000, 0, 0, 100, 0),
+(222150, 2, 3598.63, 6845.62, 140.703, 2.54014, 0, 0, 0, 100, 0),
+(222150, 3, 3587.26, 6853.75, 141.589, 2.52051, 0, 0, 0, 100, 0),
+(222150, 4, 3568.35, 6864.28, 140.533, 4.41331, 300000, 0, 0, 100, 0),
+(222150, 5, 3587.72, 6853.33, 141.546, 5.70922, 0, 0, 0, 100, 0),
+(222150, 6, 3599.61, 6845.15, 140.649, 5.70922, 0, 0, 0, 100, 0);
+
+DELETE FROM `creature` WHERE (`id1` = 22215);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1719322993021854900.sql:13]: in `creature` you delete GUID + id

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1719322993021854900.sql:13]: because GUID is unique

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1719322993021854900.sql:13]: I usually don't bother when it's a single spawn, but yes, you are right.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1719322993021854900.sql:13]: well double guid and you're fucked

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1719322993021854900.sql]: what do you mean by double guid?

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**